classification/classification_network.py

```diff
@@ -322,8 +321,10 @@ def createNetwork(args):
     dataSource   (RecordStream) Sensor region reads data from here.
     sensorType   (str)          Specific type of region, e.g. "py.RecordSensor";
                                 possible options can be found in nupic/regions/.
+    classifierType   (str)      Specific type of classifier region, e.g. "py.CLAClassifier";
```

Comment:
is there a reason why arguments are not explicitly listed? that would be cleaner and would lead to less confusion